exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/AggregateByLabelMetricTimeSeriesBuilder.java

```diff
@@ -106,7 +106,7 @@ public void recordPoint(MetricData metric, DoubleHistogramPointData point) {
     return TimeSeries.newBuilder()
         .setMetric(mapMetric(attributes, descriptor.getType()))
         .setMetricKind(descriptor.getMetricKind())
-        .setResource(mapResource(metric.getResource(), projectId));
```

Comment:
is project Id no longer required?

Reply:
It's a good question.  I did not see us using it in the exporter resources, so I assumed this was fine to do.  Maybe @aabmass knows more

Reply:
It is implicitly the same as the `name` field in the request if omitted. If you do include it, it must match name anyway. So the decision was to just omit it